### PR TITLE
delete master to worker traffic in wave_greedy

### PR DESCRIPTION
**Pull request by jiatongw**

### CODE REVIEW

File: task_mapper/wave/greedy_wave/worker/child_appointment_greedy.py [hunk 1]
```diff
@@ -57,9 +58,23 @@ def prepare_global():
         ip_to_node_name[node_ip] = node_name
         node_count += 1
```
[the remark this answers is not in the document]
Comment (jiatongw) [reply task_mapper/wave/greedy_wave/worker/child_appointment_greedy.py]: ah.. yes. but I tried ip_to_node_name[node_ip] = node_name  but the order of profiler ip and node mapping are messed up. 

But when I tried ip_to_node_name[node_name] = node_ip , the order is right. So I added one additional line of code to first get the node ip to node name and then convert them...

[the remark this answers is not in the document]
Comment (jiatongw) [reply task_mapper/wave/greedy_wave/worker/child_appointment_greedy.py]: It's weird...

[the remark this answers is not in the document]
Comment (iampradiptaghosh) [reply task_mapper/wave/greedy_wave/worker/child_appointment_greedy.py]: i see. okay no problem. Just add the comments.

[the remark this answers is not in the document]
Comment (jiatongw) [reply task_mapper/wave/greedy_wave/worker/child_appointment_greedy.py]: done

**Pull request merged**